contracts/AtomicSwap.sol

```diff
@@ -1,17 +1,17 @@
-pragma solidity ^0.5.0;
+// SPDX-License-Identifier: MIT
+pragma solidity >0.5.0;
 
 import "./ERC20.sol";
```

Comment:
let's use the IERC20(interface) here instead 😄

Reply:
`import "./ERC20.sol";` was replaced by `import "@openzeppelin/contracts/token/ERC20/SafeERC20.sol";`